packages/fxa-content-server/app/scripts/templates/settings/avatar_change.mustache

```diff
@@ -9,10 +9,12 @@
 
     <nav id="avatar-options">
       <input type="file" id="imageLoader" name="imageLoader" accept="image/png, image/jpeg"/>
-      <a href="#" id="file">{{#t}}Upload{{/t}}</a>
+      <button type="button" id="file">{{#t}}Upload{{/t}}</button>
+
       <a href="/settings/avatar/camera" id="camera">{{#t}}Camera{{/t}}</a>
```

Comment:
We can add `role="button"` here to mimic this being read as a type of "button" for screenreaders.